### ISSUE TITLE
Not working on Mac

### ISSUE THREAD

**Issue by amit-bst** [retifrav/translating-qml#3]
I have not looked at the code but i simply imported to Qt Creator and ran but it always shows 
"Failed to load translation file, falling back to English"

I have not tested this on windows

**Comment by amit-bst** [comment]
I have tested on windows now and its not working on that too.

**Comment by retifrav** [comment]
The article linked in the `README.md` might have some more steps which you could've missed. If not, then perhaps something changed in the Qt code over the past 3 years.

**Comment by amit-bst** [comment]
Yes i saw the article as well and followed it but that also gives the same result. Yes something might have changed so i just want to let you know if you can fix it.

**Comment by amit-bst** [comment]
I am using Qt 5.12.7

**Comment by retifrav** [comment]
Yes, would be nice to check what might be wrong, but sadly I have absolutely no free time at the moment, so that's unlikely to happen in the nearest future.

**Comment by amit-bst** [comment]
Not an issue at all but just created a issue to let you and others know. Thanks

**Comment by saNidzom** [comment]
Hey @amit-bst, could you check 2 things for me?
1. In your i18n folder, what files do you have?
2.When you uncomment 23. line in [trans.cpp](https://github.com/retifrav/translating-qml/blob/master/trans.cpp#L23) what does it say? Is it the same path as to your i18n folder?

**Comment by saNidzom** [comment]
This is not an issue on the Windows machine. At least not a big issue.
First, you have to run Tools>External>Linguist>Release Translations (lrelease). That will create .qm files from .ts files that are located in the i18n folder.
Now, you have 2 options, edit code (QDir - path), or Copy/Paste i18n folder to location that's printed by
`qDebug() << dir.path();`.

Best regards, and good luck :)
